draft-ietf-rats-reference-interaction-models.md

```diff
@@ -261,11 +261,11 @@ Event Logs ('eventLogs'):
 
 : Event Logs accompany Claims by providing event trails of security-critical events in a system. The primary purpose of Event Logs is to support Claim reproducibility by providing information on how Claims originated.
 
-Reference Values ('refValues')
+Verifier Inputs ('verInputs')
 
 : *mandatory*
 
-: Reference Values as defined in {{-RATS}}. This specific type of Claims is used to appraise Claims incorporated in Evidence. For example, Reference Values MAY be Reference Integrity Measurements (RIM) or assertions that are implicitly trusted because they are signed by a trusted authority (see Endorsements in {{-RATS}}). Reference Values typically represent (trusted) Claim sets about an Attester's intended platform operational state.
+: Appraisal procedures implemented by Verifiers require certain inputs as defined in {{-RATS}}: Reference Values, Endorsements, and Appraisal Policy for Evidence. These Conceptual Messages can takes various forms. For example, Reference Values can be Reference Integrity Measurements (RIM) or assertions that are implicitly trusted because they are signed by a trusted authority (Endorsements).
```

Comment:
* s/takes/take/
* The second sentence still purports Endorsements as a kind of Reference Value.

Reply:
see comment below, failed to reply here

Reply:
https://github.com/ietf-rats-wg/draft-ietf-rats-reference-interaction-models/pull/56/files#r1799830620